project/nsw-state-library-amplify/project.json

```diff
@@ -9,7 +9,8 @@
   "allowTranscriptDownload": true,
   "gaCode": "",
   "adminEmails": [
-    "glen.crawford@reinteractive.net"
```

Comment:
Based on my understanding they have been, and they've given more people access to. I'm sure Geoffrey would be able to shine some light on this mystery.